### ISSUE TITLE
Just installed it, and ERRORS.  

### ISSUE THREAD

**Issue by GeneralAwareness** [Zyin055/Config-Presets#40]
I even updated but no dice.

[ERROR][Config-Presets] The component 'txt2img_sampling' could not be processed. This may be because you are running an outdated version of the Config-Presets extension, or you included a component ID in the custom tracked components config file that does not exist or is an invalid component. This extension will not work until this issue is resolved.
[ERROR][Config-Presets] The component 'img2img_sampling' could not be processed. This may be because you are running an outdated version of the Config-Presets extension, or you included a component ID in the custom tracked components config file that does not exist or is an invalid component. This extension will not work until this issue is resolved.

**Comment by Zyin055** [comment]
What version of the A1111 web ui are you using?

If you know how to use your browser's dev tools (F12), does the <div> with id="txt2img_sampling" exist? It should be the Sampling method dropdown.

**Comment by GeneralAwareness** [comment]
commit: 22bcc7be and I didn't find that div in it.

**Comment by Zyin055** [comment]
The version you linked should have a component with id="txt2img_sampling" in it.

Are you using an extension that changes the theme or tab names or something? Can't think of anything else.

**Comment by GeneralAwareness** [comment]
Well, I am not 100% sure how to find it as I did go in the dev but this is the first time I looked at a webui page so I can't just hover my cursor over the elements to find it.  How do I find it?

**Comment by Zyin055** [comment]
It's in there inside of the div with id="sampler_selection_txt2img". FYI you can Ctrl+F to search in that side panel.

Still not sure why you get the error. Possibly one of your extension is conflicting with it?

**Comment by GeneralAwareness** [comment]
Nope, as that was the first thing I tried as I removed them all leaving only the built-ins and yours.  I am baffled.

**Comment by Zyin055** [comment]
Did you manage to figure out how to fix your issue? Or are you still without the wonderful Config Presets extension?
